scraper.py

```diff
@@ -0,0 +1,72 @@
+﻿import pandas as pd
+from bs4 import BeautifulSoup as bs
+import urllib3
+import requests
+import json
+page = requests.get('http://www.padctn.org/services/recent-sales/')
+soup = bs(page.text, 'html.parser')
+from urllib.request import urlopen, urlretrieve, quote
+from openpyxl import Workbook
+u = urlopen(site)
```

Comment:
As the other script, probably makes sense to write this function in an `if __name__ == '__main__'`

Reply:
An `if __name__ == '__main__'` is an if statement: you have to indent the stuff under it.

Reply:
actually as a general rule, please test all your scripts to make sure they actually run before committing them.